src/components/Reports/GoalCalculator/GoalCalculator.tsx

```diff
@@ -55,14 +55,26 @@ const StyledDefaultContent = styled(Box)(({ theme }) => ({
   padding: theme.spacing(2),
 }));
 
-const StyledDrawer = styled(Box)<{ open: boolean }>(({ theme, open }) => ({
+const StyledDrawer = styled(Box)<{
```

Comment:
When you add custom props to a styled component, you also need to add a `shouldForwardProp` method to exclude those custom props. You can search the codebase for examples.